src/jit/compiler.h

```diff
@@ -8068,6 +8068,56 @@ public :
     }
     info;
 
+    // Returns true if the method being compiled returns a non-void and non-struct value.
+    // Note that lvaInitTypeRef() normalizes compRetNativeType for struct returns in a 
+    // single register as per target arch ABI (e.g on Amd64 Windows structs of size 1, 2,
+    // 4 or 8 gets normalized to TYP_BYTE/TYP_SHORT/TYP_INT/TYP_LONG; On Arm Hfa structs).
+    // Methods returning such structs are considered to return non-struct return value and
+    // this method returns true.
+    bool                compMethodReturnsNonVoidNonStructNativeType() 
```

Comment:
This is a very long name that is hard to read and parse, and I'm not sure how much the "compMethod" prefix adds to the structure and understanding. How about "returnsNativeScalarType()"? And similar for the others?